pkg/exchange/bybit/exchange.go

```diff
@@ -59,3 +61,47 @@ func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
 
 	return marketMap, nil
 }
+
+func (e *Exchange) QueryTicker(ctx context.Context, symbol string) (*types.Ticker, error) {
+	s, err := e.client.NewGetTickersRequest().Symbol(symbol).DoWithResponseTime(ctx)
+	if err != nil {
+		log.Warnf("failed to get tickers, symbol: %s, err: %v", symbol, err)
+		return nil, err
+	}
+
+	if len(s.List) != 1 {
+		log.Warnf("unexpected ticker length, exp: 1, got: %d", len(s.List))
+		return nil, fmt.Errorf("unexpected ticker lenght, exp:1, got:%d", len(s.List))
+	}
+
+	ticker := toGlobalTicker(s.List[0], s.Time.Time())
+	return &ticker, nil
+}
+
+func (e *Exchange) QueryTickers(ctx context.Context, symbols ...string) (map[string]types.Ticker, error) {
+	tickers := map[string]types.Ticker{}
+	if len(symbols) > 0 {
+		for _, s := range symbols {
```

Comment:
Is it required to sleep between api calls?

Reply:
ok, i will implement him